Plugins/Other/aman-config.json

```diff
@@ -1,19 +1,45 @@
 {
     "openAutomatically": false,
     "timelines": {
-        "19R/19L": {
+        "GM 19R/19L": {
             "targetFixes": [ "GSW40", "GME40" ],
-            "tagLayout": "myLayout",
+            "viaFixes": [ "ADOPI", "LUNIP", "ESEBA", "INREX", "RIPAM", "BELGU" ],
+            "tagLayout": "DualrunwayLayout",
             "destinationAirports": [ "ENGM" ]
         },
-        "01L/01R": {
+        "GM 01L/01R": {
             "targetFixes": [ "OBW40", "ONE40" ],
-            "tagLayout": "myLayout",
+            "viaFixes": [ "ADOPI", "LUNIP", "ESEBA", "INREX", "RIPAM", "BELGU" ],
+            "tagLayout": "DualrunwayLayout",
             "destinationAirports": [ "ENGM" ]
+        },
+        "BR17": {
+            "targetFixes": [ "BR04" ],
+            "viaFixes": [ "NIDGI", "LUNUR", "ETNOR", "PESUR", "MODNI", "GENVU", "ALUVA", "BADAB" ],
+            "tagLayout": "SinglerunwayLayout",
+            "destinationAirports": [ "ENBR" ]
+        },
+        "BR35": {
+            "targetFixes": [ "BG04" ],
+            "viaFixes": [ "NIDGI", "LUNUR", "ETNOR", "PESUR", "MODNI", "GENVU", "ALUVA", "BADAB" ],
+            "tagLayout": "SinglerunwayLayout",
+            "destinationAirports": [ "ENBR" ]
+        },
+        "VA 09": {
+            "targetFixes": [ "TO03" ],
+            "viaFixes": [ "VEVOD", "NELSU", "NUPGO", "MISVO" ],
+            "tagLayout": "SinglerunwayLayout",
+            "destinationAirports": [ "ENVA" ]
+        },
+        "VA 27": {
+            "targetFixes": [ "TD48" ],
```

Comment:
Kan targetFixes på flyplassene være ICAO kodene?

Husker ikke grunnen til at vi ikke kunne/burde gjør dette på GM, men har noe med dual rwy's. Ser ikke problemet med å gjør det på BR og VA

Reply:
Kan teste dette, gi meg et øyeblikk. Forresten er det mer hensiktsmessig å ha targetfix på FAP eller nærmest THR/flyplassen?

Reply:
Å bruke ICAO koden vil gjør at AMAN fortsatt fungerer på feks RNP approacher.

Reply:
> Å bruke ICAO koden vil gjør at AMAN fortsatt fungerer på feks RNP approacher.

Da gjør vi det, tester det straks og kommer med update

Reply:
![image](https://user-images.githubusercontent.com/79529337/192564434-f6a5b32b-78c0-4186-87e7-65b1c00e999d.png)
Dupe i så fall men det funker. Merger til kun en config for ENVA i så fall om du er enig @Adrian2k